pkg/mock/sdkserver.go

```diff
@@ -14,10 +15,11 @@ import (
 // SdkServers consists of different mock servers that the mock
 // sdk server can implement
 type SdkServers struct {
-	Cluster *MockOpenStorageClusterServer
-	Node    *MockOpenStorageNodeServer
-	Role    *MockOpenStorageRoleServer
-	Volume  *MockOpenStorageVolumeServer
+	Cluster        *MockOpenStorageClusterServer
+	Node           *MockOpenStorageNodeServer
+	Role           *MockOpenStorageRoleServer
+	Volume         *MockOpenStorageVolumeServer
+	PortworxServer *MockPortworxServiceServer
```

Comment:
nit: Let's rename this to `PortworxService` to be slightly consistent with the other names.